src/components/icon-button/icon-button.tsx

```diff
@@ -72,10 +85,12 @@ export class IconButton {
 
         this.mdcIconButtonRipple = new MDCRipple(element);
         this.mdcIconButtonRipple.unbounded = true;
+        this.host.addEventListener('click', this.removeFocusedStateOnClick);
```

Comment:
Since activating the button by pressing enter on a keyboard creates a synthetic `click` event, this removes the styling even if the user presses enter, instead of clicking. That's not optimal, but I think we can live with it. If the user interacts with the element, they hopefully know "where they are", despite the visual style disappearing.

The actual focus isn't removed, just the styling. (Which, by the way, makes me want to rename `removeFocusedStateOnClick` to `removeFocusedStyleOnClick`. I think I'll do that.)